test/functional/provider/nodejs_spec.lua

```diff
@@ -61,8 +66,7 @@ describe('nodejs host', function()
       nvim.command('call jobstop(g:job_id)');
     ]])
     command('let g:job_id = jobstart(["node", "'..fname..'"])')
-    sleep(5000)
-    eq('hello-plugin', eval('g:job_out'))
-    os.remove(fname)
+    sleep(500)
+    retry(nil, 4500, function() eq('hello-plugin', eval('g:job_out')) end)
```

Comment:
https://ci.appveyor.com/project/neovim/neovim/build/17222/job/xsuglue2xs24b8rs

```
[00:07:47] [ RUN      ] nodejs can inspect: 156.00 ms OK
[00:07:48] [ RUN      ] nodejs host works: 671.00 ms OK
[00:07:49] [ RUN      ] nodejs host plugin works: 532.00 ms OK
```

I guess `sleep(500)` isn't necessary after all. Each test should be done within 1 second.